subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CopyTaskIntegrationSpec.groovy

```diff
@@ -2197,7 +2207,6 @@ class CopyTaskIntegrationSpec extends AbstractIntegrationSpec {
         method << ["from", "into"]
     }
 
-    @Unroll
```

Comment:
Shouldn't we remove these in one go?

Reply:
What do you mean in one go? In all of our code base?

Reply:
Sorry for not being more clear, yes, that's what I mean. Like it only adds noise to remove them as part of PRs.